src/modules/landing/components/reviews-card/review-cards.tsx

```diff
@@ -0,0 +1,61 @@
+import { type FC } from 'react';
+import Image from 'next/image';
+// components
+import { Typography } from '@/modules/core/components/typogrpahy';
+// types
+import type { ReviewCardProps } from './review-cards.interface';
+import img from '@/assets/images/girl.png';
+
+export const ReviewCard: FC<ReviewCardProps> = ({
+  text,
+
+  name,
+  occupation,
+}) => {
+  return (
+    <div className=' p-4 border-2 rounded-[12px] border-[#F9F9F9] flex flex-col items-center xl:flex-row gap-6 '>
```

Comment:
There a lot of unnecessary spaces at the start and end of the string, please remove them in whole file. And make a double check in other files.